client/app/dedicated/server/bandwidth/public-cancel/public-cancel.controller.js

```diff
@@ -0,0 +1,34 @@
+export default class {
+  /* @ngInject */
+  constructor($rootScope, $scope, $translate, Server, Alerter) {
+    this.setMessage = $scope.setMessage;
+    this.$rootScope = $rootScope;
+    this.$translate = $translate;
+    this.Server = Server;
+    this.Alerter = Alerter;
+  }
+
+  $onInit() {
+    this.loader = {
```

Comment:
IMO if we only have one loader we could avoid using an object of loader flag.

Reply:
It was from the legacy, I'll take a look if this could be improved